refinery/templates/core/workflow.html

```diff
@@ -74,12 +74,14 @@ <h4>Layout</h4>
 		<p>
 			<label class="radio">
                 <input type="radio" name="optionsRadios" id="cb_layout_kind_refinery" value="1"
-                       onclick="workflowVisualizationModule.reload_workflow('{% url 'download_workflow' workflow.uuid %}')"
+                       onclick="workflowVisualizationModule.reload_workflow(
+                         '{% url 'workflow-download-workflow' workflow.uuid %}')"
```

Comment:
Sorry, but I don't see where this is coming from: The name seems odd, but maybe it's the best choice in context?

Reply:
Ahh so Django Rest framework provides human readable names for each url within a Router.
In this case for a detail route in `WorkflowViewSet` its: `{basename}-{methodname}`

This link was helpful to me: http://www.django-rest-framework.org/api-guide/routers/#defaultrouter

`{% url 'workflow-download-workflow' workflow.uuid %}` ends up becoming:
`/api/v2/workflows/<workflow_uuid>/download_workflow`

Looking at this again, I think `/api/v2/workflows/<workflow_uuid>/download` would be better